### DISCUSSION TITLE
Toasts

### DISCUSSION THREAD

**Discussion by Courtny** [microsoft/atlas-design#178] [Component]
Floating notices, but aligned to the top and center of the page and they disappear after a set time.

**Comment by meredithxie1995** [comment]
![Screen Shot 2021-05-17 at 4 52 17 PM](https://user-images.githubusercontent.com/84340484/118570103-40e51400-b730-11eb-9349-9067a3e3ad0b.png)
This toast shows up about 5 seconds (we need to investigate on how long exactly does the toast shows up before disappear). There is a link to the location of the object on the toast.
